Examples/PipelineExample.md

```diff
@@ -0,0 +1,42 @@
+# Pipeline 
+## An example of pipelines Redis Stack Redis commands (JSON.SET & JSON.CLEAR & JSON.GET)
+
+### Connect to the Redis server:
+```csharp
+var redis = ConnectionMultiplexer.Connect("localhost");
+```
+
+### Setup pipeline connection
+```csharp
+var pipeline = new Pipeline(ConnectionMultiplexer.Connect("localhost"));
```

Comment:
I added another option to Pipeline and it can also receive an instance of Idatabase in this way:
```csharp
var redis = ConnectionMultiplexer.Connect("localhost");
IDatabase db = redisFixture.Redis.GetDatabase();
var pipeline = new Pipeline(db);
```
We should add an example to this as well.

Reply:
```suggestion
        // Connect to the Redis server
        var redis = ConnectionMultiplexer.Connect("localhost");

        // Get a reference to the database
        var db = redis.GetDatabase();
        var ts = db.TS();

        // Setup pipeline connection
        var pipeline = new Pipeline(redis);

```
added in other example

Reply:
the line `var ts = db.TS();` should be after, and you can write 
```csharp
var pipeline = new Pipeline(db);
```
 with `db` instead of `redis`